add_wasm.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"syscall/js"
+)
+
+// Add two numbers.
+func Add(a int, b int) int {
+	return a + b
+}
+
+// Gets a HTML element given an ID.
+func getElementById(id string) js.Value {
+	return js.Global().Get("document").Call("getElementById", id)
+}
+
+// Gets a numeric value from an HTML element body given an ID.
+func getNumberFromElement(id string) (int, error) {
+	element := getElementById(id)
+	valueStr := element.Get("value").String()
+	return strconv.Atoi(valueStr)
+}
+
+// Adds the two input elements and sets the sum to another element.
+func AddAndSetResult() {
+	sumNode := getElementById("sum")
+	number1, err := getNumberFromElement("number1")
+	if err != nil {
+		sumNode.Set("innerHTML", err.Error())
+		fmt.Println(err)
+		return
+	}
+	number2, err := getNumberFromElement("number2")
+	if err != nil {
+		sumNode.Set("innerHTML", err.Error())
+		fmt.Println(err)
+		return
+	}
+	sumNode.Set("innerHTML", Add(number1, number2))
+}
+
+// Start of execution.
```

Comment:
minor: do these comments add value ?

Reply:
It's just the Go template for function doc comments. Keeping it in lieu of format. No functional value in this case.